Userland/Libraries/LibWeb/HTML/HTMLLinkElement.idl

```diff
@@ -10,7 +10,7 @@ interface HTMLLinkElement : HTMLElement {
     [CEReactions, Reflect] attribute DOMString href;
     [FIXME, CEReactions] attribute DOMString? crossOrigin;
     [CEReactions, Reflect] attribute DOMString rel;
-    [FIXME, CEReactions] attribute DOMString as;
+    [CEReactions, Reflect] attribute DOMString as;
```

Comment:
Hmm, doesn't using `Reflect` mean that your `as()` implementation won't even get called? (rather we just access the attribute directly)

Please add a test that confirms the expected behaviors :)

Reply:
that was correct mb should be fixed now